testing/python_lib/integration_base.py

```diff
@@ -141,6 +144,40 @@ def _get_faucet_config_path(self):
         return os.path.dirname(os.path.abspath(__file__)) + \
             (config_file_format % ('forch-faucet-1'))
 
+    def parallelize(self, job_count, target, target_args=None):
+        """Parallelizes multiple runs of a target method with multiprocessing"""
+        jobs = []
+        if not target_args:
+            target_args = []
+        for job in range(job_count):
+            try:
```

Comment:
rather than catching the exception, do an explicit if check with the length

Reply:
Changed